modules/ding_loan/ding_loan.module

```diff
@@ -48,8 +48,8 @@ function ding_loan_sort_expiry($a, $b) {
  */
 function ding_loan_ding_session_cache_defaults() {
   return array(
-    'titel' => 'Ding loans',
+    'title' => 'Ding loans',
     'enabled' => TRUE,
-    'expire' => 3600,
+    'expire' => 60,
```

Comment:
This reduces the loan cache from an hour to a minute. That seems pretty drastic - and also unrelated to the change at hand. Why is this?